src/main/config/wikipedia-stats.properties

```diff
@@ -40,7 +39,6 @@ serializers.registry.integer.class=org.apache.samza.serializers.IntegerSerdeFact
 
 # Systems
 systems.kafka.samza.factory=org.apache.samza.system.kafka.KafkaSystemFactory
-systems.kafka.samza.msg.serde=json
 systems.kafka.consumer.zookeeper.connect=localhost:2181/
 systems.kafka.consumer.auto.offset.reset=largest
 systems.kafka.producer.bootstrap.servers=localhost:9092
```

Comment:
Would be good to change the stores to tables in this example and use table descriptors. Can do separately if it's too much work.